src/System.Data.SqlClient/tests/Tools/TDS/TDS.EndPoint/Interop.Libraries.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why is this file being copied?  If you need access to, you can include it from Common.